File: .claude/tmp/run_gh.py
import subprocess
import sys

GH = r"C:\Program Files\GitHub CLI\gh.exe"
OUT = r"C:\projets\IA\app_quizz_claude\.claude\tmp\gh_output.txt"

def run(args):
    try:
        r = subprocess.run([GH] + args, capture_output=True, text=True, timeout=30)
        return f"STDOUT: {r.stdout}\nSTDERR: {r.stderr}\nRC: {r.returncode}\n"
    except Exception as e:
        return f"EXCEPTION: {e}\n"

out = ""
out += "=== VERSION ===\n" + run(["--version"])
out += "=== AUTH STATUS ===\n" + run(["auth", "status"])
out += "=== ISSUE LIST ===\n" + run(["issue", "list", "--repo", "gaetan-perso/claude-quizz", "--limit", "50", "--state", "open", "--json", "number,title"])

with open(OUT, 'w') as f:
    f.write(out)

print("Done - wrote to", OUT)

File: .claude/tmp/run_gh.js
const { execSync } = require('child_process');
const fs = require('fs');
const out_file = 'C:/projets/IA/app_quizz_claude/.claude/tmp/gh_output.txt';

function run(cmd) {
  try {
    return execSync(cmd, { encoding: 'utf8', timeout: 30000 });
  } catch (e) {
    return 'ERROR: ' + e.message + '\nSTDOUT: ' + (e.stdout || '') + '\nSTDERR: ' + (e.stderr || '');
  }
}

const GH = '"C:\\Program Files\\GitHub CLI\\gh.exe"';

let output = '';
output += '=== GH VERSION ===\n';
output += run(`${GH} --version`) + '\n';

output += '=== GH AUTH STATUS ===\n';
output += run(`${GH} auth status`) + '\n';

fs.writeFileSync(out_file, output);
console.log('Done');
